### PR TITLE
feat: Companies component

### PR DESCRIPTION
**Pull request by amenes12**
### What does it do

Created companies' component.

### Why is it needed

Manage companies' information with CRUD actions.

### Future tasks

- [x]  Improve UI

### CODE REVIEW

File: src/app/employers/pages/companies/companies.component.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      console.log("sending update...");
+      this.updateCompany(company.id, company);
+    } else {
+      console.log("sending create...");
+      this.createCompany(company);
+    }
+    this.cancelEdit();
+    console.log("finished...");
+  }
+}
```
Comment (dalbitresb12) [inline src/app/employers/pages/companies/companies.component.ts:79]: Can you remove all `console.log`?

**Review by dalbitresb12 — CHANGES_REQUESTED** [review]
Minor stuff, looks mostly good.

File: src/app/employers/pages/companies/companies.component.ts (hunk 1 above)
Comment (amenes12) [reply src/app/employers/pages/companies/companies.component.ts]: Done

**Review by dalbitresb12 — APPROVED** [review]
Aweomse, thanks!

**Pull request merged**